projects/libvpx/build.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash -eu
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# Build libvpx
+build_dir=$WORK/build
+mkdir -p ${build_dir}
+pushd ${build_dir}
+
+export CFLAGS="$CFLAGS -fno-sanitize=vptr"
+export CXXFLAGS="$CXXFLAGS -fno-sanitize=vptr"
+
+# Remove files generated by the previous build.
+rm -rf ./*
+
+LDFLAGS="$CXXFLAGS" $SRC/libvpx/configure \
+    --disable-unit-tests \
+    --size-limit=12288x12288 \
+    --extra-cflags="-DVPX_MAX_ALLOCABLE_MEMORY=1073741824" \
+    --target="generic-gnu"
```

Comment:
we could do both builds, but to start I'd prefer testing the optimized one to reduce timeouts and give more real world coverage even if it's only for x86 targets.

Reply:
Agreed. Will remove generic-gnu target. AV1 uses generic target when memory sanitizer is enabled.

Reply:
In other projects we enable asserts (--enable-debug) when fuzzing to make sure no 'impossible' situations occur.

Reply:
sure will add.

Reply:
Made the required changes in latest commit 3171ccac249ecd4f0877ed8ec9cdad00ec1e2e7b